src/components/Header/Header.module.scss

```diff
@@ -86,6 +87,34 @@
   }
 }
 
+.skipToContent {
```

Comment:
I personally see this more as a "general" accessibility feature than specifically a header feature. Therefore I find `accessibility.scss` more logical. I understand this is debatable, based on how you look at it.

I think it's ok to leave it as-is. It's a minor, but I wanted to share it with you. Based on our custom styling approach for tier 2/3, it could be that this eventually should be moved. But let's first wait before that happens ;)

Reply:
Since it is solely used in the `Header` and has specific styling applied for the header, I placed it there. If we ever need to change it because of a custom styling approach, we can tackle it then!